### PR TITLE
Feature create sample nfts

### PR DESCRIPTION
**Pull request by danielcdz**
## Background
- This PR introduces a smart contract that mints sample NFTs and stores them in a Sepolia smart contract.

## Summary
- Add ERC721 open-zeppelin smart contract.
- Add required assests.

### CODE REVIEW

File: packages/snfoundry/contracts/src/SepolliaNFT.cairo [hunk 1]
```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+// Compatible with OpenZeppelin Contracts for Cairo ^0.13.0
+
+#[starknet::contract]
+mod SepolliaNFT {
+    use openzeppelin::access::ownable::OwnableComponent;
+    use openzeppelin::introspection::src5::SRC5Component;
+    use openzeppelin::token::erc721::ERC721Component;
+    use openzeppelin::token::erc721::ERC721HooksEmptyImpl;
```
Comment (wugalde19) [inline packages/snfoundry/contracts/src/SepolliaNFT.cairo:9]: As specified [here](https://youtu.be/3vn8tgHYKqI?si=41oTI8hnxlCc5gOl&t=44) you need to get rid of this line

**Review by wugalde19 — CHANGES_REQUESTED** [review]

File: packages/snfoundry/contracts/src/SepolliaNFT.cairo
[the remark this answers is not in the document]
Comment (danielcdz) [reply packages/snfoundry/contracts/src/SepolliaNFT.cairo:1]: sure!

[the remark this answers is not in the document]
Comment (danielcdz) [reply packages/snfoundry/contracts/src/SepolliaNFT.cairo:1]: done! ✅

File: packages/snfoundry/contracts/src/SepolliaNFT.cairo (hunk 1 above)
Comment (danielcdz) [reply packages/snfoundry/contracts/src/SepolliaNFT.cairo]: I thought I deleted that line hehe, it is deleted now! ✅

File: packages/snfoundry/contracts/src/SepoliaNFT.cairo [hunk 2]
```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
+// Compatible with OpenZeppelin Contracts for Cairo ^0.13.0
+
+#[starknet::contract]
+mod SepolliaNFT {
```
Comment (wugalde19) [inline packages/snfoundry/contracts/src/SepoliaNFT.cairo:5]: Please rename it to `SepoliaNFT`

File: packages/snfoundry/contracts/src/lib.cairo [hunk 3]
```diff
@@ -1,4 +1,5 @@
 mod YourContract;
+mod SepolliaNFT;
```
Comment (wugalde19) [inline packages/snfoundry/contracts/src/lib.cairo:2]: Please rename it to `SepoliaNFT`

File: packages/snfoundry/contracts/src/SepoliaNFT.cairo [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        SRC5Event: SRC5Component::Event,
+        #[flat]
+        OwnableEvent: OwnableComponent::Event,
+        #[flat]
+        UpgradeableEvent: UpgradeableComponent::Event,
+    }
+
+    #[constructor]
+    fn constructor(ref self: ContractState, owner: ContractAddress) {
+        self.erc721.initializer("SepolliaNFT", "SNFT", "https://ipfs.io/ipfs/QmYRSjVsppta8PhuqpsnTeb4qcWBgLCZYcsEgSBkPzsJGu/");
```
Comment (wugalde19) [inline packages/snfoundry/contracts/src/SepoliaNFT.cairo:55]: Please rename it to `SepoliaNFT`

**Review by wugalde19 — CHANGES_REQUESTED** [review]

File: packages/snfoundry/contracts/src/SepoliaNFT.cairo (hunk 4 above)
Comment (danielcdz) [reply packages/snfoundry/contracts/src/SepoliaNFT.cairo]: Done! ✅

File: packages/snfoundry/contracts/src/lib.cairo (hunk 3 above)
Comment (danielcdz) [reply packages/snfoundry/contracts/src/lib.cairo]: Done! ✅

File: packages/snfoundry/contracts/src/SepoliaNFT.cairo (hunk 2 above)
Comment (danielcdz) [reply packages/snfoundry/contracts/src/SepoliaNFT.cairo]: Done! ✅

**Review by wugalde19 — APPROVED** [review]

**Pull request merged**